pycbc/workflow/inference_followups.py

```diff
@@ -769,6 +862,27 @@ def make_posterior_workflow(workflow, samples_files, config_file, label,
     return posterior_file, summary_files, prior_plots, posterior_plots
 
 
+def add_inference_workflow_group(parser):
```

Comment:
This doesn't seem inference specific.

Reply:
This seems to be pretty much common to a lot of the workflows? Can we reduce the redundancy here?

Reply:
I don't know. I looked at some of the other workflows, like `coinc_search`, `psd_estimation`, and didn't see the same set of options. Those workflows just take a workflow name and output directory; none of the `dax-filename` `transformation-catalog` options are used. The only one that looks similar is `sbank_workflow`, although some of the options names and defaults are different there. I don't want to touch the other workflows. I suppose I could get rid of these extra options from the inference workflows to make them more like `coinc_search`. Honestly, I don't know what the options do exactly. I think @cmbiwer 's initial intent was to have the inference workflow be something that could run as a sub-workflow of the search, in which case those options are necessary (I think).

Reply:
I'm with Alex on this one. Perhaps the add_workflow_group could be sent a `can_be_subworkflow` flag, which would add the options needed to specify all the filenames. But this does seem like a set of options that *all* workflows use / should have.

Reply:
@ahnitz @spxiwh Ok, I've moved the function to add the workflow options to `workflow/core.py` and called it `add_workflow_settings_cli` (so as not to conflict with the `add_workflow_command_line_group` in `configuration.py`, which is a bit confusingly named since it only applies to config files, yet all of these functions are promoted to the `pycbc.workflow` namespace).  I also made the sub-dax arguments optional, as @spxiwh suggested. I've updated the three inference workflows to point to this function. I did *not* touch any of the other workflows, as I'm not certain how to test them. If you two would like to update the other workflows to point to this function after this PR is merged, feel free.